docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
 # Setting up your simulation from scratch
 
-## Hydrostatic tank
+In this tutorial, we will guide you through the general structure of simulation files.
+We will set up a simulation similar to the example simulation
+[`examples/fluid/hydrostatic_water_column_2d.jl`](https://github.com/trixi-framework/TrixiParticles.jl/blob/main/examples/fluid/hydrostatic_water_column_2d.jl),
+which is one of our simplest example simulations.
+In the second part of this tutorial, we will show how to replace components
+of TrixiParticles.jl by custom implementations from within a simulation file,
+without ever cloning the repository.
 
+For different setups and physics, have a look at [our other example files](@ref examples).
```

Comment:
Maybe explain how we usually structure our example files. Resolution, experiment setup, ...

Reply:
Yes I agree this doesn't belong here.